src/test/java/com/github/krzysiekjodlowski/ox/ConsoleUITest.java

```diff
@@ -46,19 +46,19 @@ public void testGettingNumberFromUserWhileNumberInRange(
         //assert
         assertEquals(output, expected);
     }
-
-    @Test(dataProvider = "arbitraryNumbersInputNotInTheRange"
-            , expectedExceptions = NumberOutOfRangeException.class)
-    public void testThrowsNumberOutOfRangeExceptionWhileNumberInRange(
-            String input, NumberRange<Integer> numberRange)
-            throws NumberOutOfRangeException {
-        //arrange
-        Scanner scanner = new Scanner(input);
-        UI<String, Integer> consoleUI = new ConsoleUI(scanner, new PrintStream(System.out));
-
-        //act
-        consoleUI.getNumberFromUser(numberRange);
-
-        //assert throws
-    }
+//
+//    @Test(dataProvider = "arbitraryNumbersInputNotInTheRange"
+//            , expectedExceptions = NumberOutOfRangeException.class)
+//    public void testThrowsNumberOutOfRangeExceptionWhileNumberInRange(
+//            String input, NumberRange<Integer> numberRange)
+//            throws NumberOutOfRangeException {
+//        //arrange
+//        Scanner scanner = new Scanner(input);
+//        UI<String, Integer> consoleUI = new ConsoleUI(scanner, new PrintStream(System.out));
+//
+//        //act
+//        consoleUI.getNumberFromUser(numberRange);
+//
+//        //assert throws
+//    }
```

Comment:
Remember to uncomment or delete. You can create issue for this.

Reply:
Deleted for now.